src/Models/Media.php

```diff
@@ -194,6 +194,25 @@ public function getMediaConversionNames(): array
         })->toArray();
     }
 
+    public function hasGeneratedConversion(string $conversionName): bool
+    {
+        $generatedConversions = $this->getGeneratedConversions();
+
+        return $generatedConversions[$conversionName] ?? false;
```

Comment:
That's not true, because `$generatedConversions[$conversionName]` can contain `true` or `false`.

So if `$generatedConversions[$conversionName]` is set, it will return either `true` or `false`, if it's not set the fallback is `false`.

Reply:
Whoops, my bad 👍

Reply:
We could use `array_get` to make this a one liner. But unsure if that would be better.